orchestra/actions/util/__init__.py

```diff
@@ -0,0 +1,69 @@
+from collections import OrderedDict
+
+from .impl import _get_script_output
+from .impl import _get_subprocess_output
+from .impl import _run_internal_script
+from .impl import _run_internal_subprocess
+from .impl import _run_user_script
+
+
+def run_internal_script(script, environment: OrderedDict = None):
+    """Helper for running internal scripts.
+    If the script returns a nonzero exit code an OrchestraException is raised.
+    :param script: the script to run
+    :param environment: optional additional environment variables
+    :return: a subprocess.CompletedProcess instance
```

Comment:
Why return the `CompletedProcess`? This opens callers to do custom things, which is what this PR should try to avoid.
More simple helper that do one obivous thing.